docs/userguides/logparam.md

```diff
@@ -109,6 +109,9 @@ variables in run-time, but note the following:
     logging framework.
 -   The reading or writing of a parameter can be done at any time once
     you are connected to the Crazyflie.
+-   If the callback parameter version is used to get notified that it has
+    changed, it will run from the param task, it should not block and should 
+    run quickly.
 
```

Comment:
I am having trouble parsing this, what does _version_ mean it this context?

ah, I think I got it, perhaps:

"If the `PARAM_ADD_WITH_CALLBACK` macro is used [...]" or "if the `PARAM_CALLBACK`type is set [...]"